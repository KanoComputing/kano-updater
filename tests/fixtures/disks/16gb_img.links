/dev/disks/by-label/boot -> ../../mmcblk0p1
/dev/disks/by-label/rootfs -> ../../mmcblk0p2
