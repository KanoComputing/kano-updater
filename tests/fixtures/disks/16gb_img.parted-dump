Model: SD SB16G (sd/mmc)
Disk /dev/mmcblk0: 31116288s
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start    End        Size       Type     File system  Flags
 1      8192s    409600s    401409s    primary  fat16        lba
 2      425984s  11698175s  11272192s  primary  ext4

