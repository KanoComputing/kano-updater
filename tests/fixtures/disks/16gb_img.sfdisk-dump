{
    "partitiontable": {
        "label": "dos",
        "id": "0x000198c8",
        "device": "/dev/mmcblk0",
        "unit": "sectors",
        "partitions": [
                    {"node": "/dev/mmcblk0p1", "start": 8192, "size": 409598, "type": "c"},
                    {"node": "/dev/mmcblk0p2", "start": 3866624, "size": 27249664, "type": "83"}
        ]
    }
}
