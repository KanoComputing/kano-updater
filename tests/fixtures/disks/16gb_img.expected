{
    "commands": [
        "parted unit s resizepart 2 31116287",
        "partprobe /dev/mmcblk0",
        "resize2fs /dev/mmcblk0p2"
    ]
}
